samples/client/petstore/powershell-experimental/src/PSPetstore/Model/Pet.ps1

```diff
@@ -64,6 +64,14 @@ function Initialize-PSPet {
         'Creating PSCustomObject: PSPetstore => PSPet' | Write-Debug
         $PSBoundParameters | Out-DebugParameter | Write-Debug
 
+        if (!$Name) {
```

Comment:
We can use the Powershell attribute at the parameter declaration level, 
[ValidateNotnull()]
[$string]$Name ,

Pattern can be checked : [ValidatePattern("Pattren")]

more details can be found :- https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7

Reply:
Ok. Will look into that.

Reply:
Done.